tests/model_update_test.js

```diff
@@ -0,0 +1,118 @@
+require('./helper');
+
+var expect = require('chai').expect;
+var Sequelize = require('..');
+
+const DataTypes = Sequelize.DataTypes;
+const sinon = require('sinon');
```

Comment:
sinon isn't added as a dep in package.json

Reply:
seems like I forgot, I will add then